src/modules/thirdParty/types.ts

```diff
@@ -5,4 +5,5 @@ export type ThirdParty = {
   description: string
   maxItems: string
   totalItems: string
+  availableSlots?: number
```

Comment:
Question!
Does it make sense to have `availableSlots` as a `number | undefined`? couldn't it be zero if it's not defined?

Again, not sure, just asking

Reply:
Mhmm.. I'm not sure neither. 
I think `availableSlots?: number` is transformed to `availableSlots: number | undefined` tho, so it would be the same.

Reply:
thanks for the suggestion Nico! I've tried that and as you said, it gets a bit difficult to know if the `avaiableSlots` is truly 0 or if they weren't fetch yet. I'm merging it for now, we can come back later to this if we need :)